Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -145,25 +139,35 @@ public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationT
                     }
                     catch (SerializationException ex)
                     {
-                        const string LogTemplate =
-                            "Failed to deserialize the plugin manifest retrieved from {PluginManifestUrl}. If you " +
-                            "have specified a custom plugin repository manifest URL with --plugin-manifest-url or " +
-                            PluginManifestUrlKey + ", please ensure that it is correct.";
-                        _logger.LogError(ex, LogTemplate, manifestUrl);
-                        throw;
+                        _logger.LogError(ex, "Failed to deserialize the plugin manifest retrieved from {Manifest}", manifest);
+                        return Enumerable.Empty<PackageInfo>();
                     }
                 }
             }
             catch (UriFormatException ex)
             {
-                const string LogTemplate =
-                    "The URL configured for the plugin repository manifest URL is not valid: {PluginManifestUrl}. " +
-                    "Please check the URL configured by --plugin-manifest-url or " + PluginManifestUrlKey;
-                _logger.LogError(ex, LogTemplate, manifestUrl);
-                throw;
+                _logger.LogError(ex, "The URL configured for the plugin repository manifest URL is not valid: {Manifest}", manifest);
+                return Enumerable.Empty<PackageInfo>();
+            }
+            catch (HttpException ex)
+            {
+                _logger.LogError(ex, "An error occurred while accessing the plugin manifest: {Manifest}", manifest);
+                return Enumerable.Empty<PackageInfo>();
             }
         }
 
+        /// <inheritdoc />
+        public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationToken cancellationToken = default)
+        {
+            var result = new List<PackageInfo>();
+            foreach (RepositoryInfo repository in _config.Configuration.PluginRepositories)
```

Comment:
there should be an "official" vs "unofficial" handling... could be as easy as presence of `github.com/jellyfin/` substring in manifest URL

Reply:
Seems like that would be a frontend thing. What would you propose happens with unofficial repos here?

Reply:
Frontend is an untrusted place, same about deciding when to autoupdate vs when not.
Also frontend should just have a flag to show "official" vs "3rd party" IMHO.

So I think a third field defined by server is needed (whether the repo is official or not).

Reply:
This isn't really an issue we can solve though, because clients will always be able to ignore the flag when looking at repository. Since they have absolute power over which repositories are added, we can't do anything about unofficial repositories.

Reply:
It seems I worded my concern poorly.

What I suggest is more like server sending the client an `official`/`3rd party` flag so client can show it, otherwise to update the logic that determines this state we'd have to update all clients. So it's more about central part of determining what to show than about security.

Reply:
Discussed internally. Decided to leave this out and instead (in the future) provide a way to enable/disable per-plugin autoupdate (with defaults potentially being different for official vs unofficial plugins), though there is no big need currently to send "official" status to frontend.